sale_purchase_order_line/security/security.xml

```diff
@@ -0,0 +1,27 @@
+<odoo>
+
+    <data noupdate="0">
+        <record model="ir.module.category" id="sale_purchase_order_line_module">
+            <field name="name">Extra permissions</field>
+            <field name="description">Access for sales and purchases order lines</field>
+            <field name="sequence">10</field>
+        </record>
+
+        <record id="group_extra_permission" model="res.groups">
+            <field name="name">Extra Permisisions</field>
+            <field name="category_id" ref="sale_purchase_order_line_module"/>
+            <field name="implied_ids" eval="[(4, ref('sales_team.group_sale_salesman')),(4, ref('purchase.group_purchase_user'))]"/>
+            <field name="users" eval="[(4,ref('base.user_admin'))]"/>
+        </record>
```

Comment:
This wasn't what I wanted to say... But this:
* Create a new group "Sale Purchase order line: can see report"
* Place it under "Extra permissions" (in spanish "permisos extra") existing category (which XML ID is 	`base.module_category_usability`).
* Implied groups are right